src/hooks/useInlineErrors.test.ts

```diff
@@ -1,12 +1,12 @@
-import { renderHook } from "@testing-library/react";
+import { act, renderHook } from "@testing-library/react";
 
 import useInlineErrors from "./useInlineErrors";
 
 describe("useInlineErrors", () => {
   it("can add an error", () => {
     const { result, rerender } = renderHook(() => useInlineErrors());
     let [errors, setError] = result.current;
-    setError("Error1", "Uh oh!");
+    act(() => setError("Error1", "Uh oh!"));
     rerender();
     [errors] = result.current;
```

Comment:
Looks like I somehow doubled up the rerender and destructure in this test so you could remove these two lines.

Reply:
Removed this part.